multimedia/templates/multimedia/book_list.html

```diff
@@ -0,0 +1,19 @@
+{% extends "multimedia/book_base.html" %}
+{% load staticfiles %}
+
+{% block content %}
+{{ block.super }}
```

Comment:
Which part is not clear?

I'm not sure if this template is still used or will be further develop.

The name convention is quite intuitive. It should be listing books.